ons_alpha/core/static/js/previous-version-block.js

```diff
@@ -0,0 +1,49 @@
+/* global $ */
+
+class PreviousVersionBlock {
+    constructor(blockDef, placeholder, prefix, initialState) {
+        this.blockDef = blockDef;
+        this.element = document.createElement('div');
+
+        placeholder.replaceWith(this.element);
+
+        this.renderContent(initialState);
+    }
+
+    renderContent(state) {
+        if (state) {
+            this.element.innerHTML = `
+                <a class="button button-small button-secondary" href="../revisions/${state}/view/">View revision</a>
+                <a class="button button-small button-secondary" href="../revisions/${state}/revert/">Review revision</a>
```

Comment:
suggestion: either 
a. drop "Review revision" as when go to is, the actions are destruction - in that they promote that revision as the latest 
b. replace the action to our own that saves the revision in place